### ISSUE TITLE
question

### ISSUE THREAD

**Issue by deathcult456** [JRB-90/jpower#4]
hello, your project is super interesting and I would like to be able to participate in it but I lack information.
1: Do you use a XIAO BLE SENSE?
2: which program you use to compile/transfer the code
3: I mainly use the arduino IDE do you think you can convert it?
Thanks a lot!

**Comment by JRB-90** [comment]
Hi, this is currently a personal project and is work in progress. I am just hosting it here on github as a way to version control it, it is far from being finished. When it is completed to an acceptable level, I shall put proper instructions into the README, but for now, I am changing things on a weekly basis, so I am won't be documenting anything yet.

In answer to your questions...

1) I used a XIAO BLE SENSE for prototyping originally. But it had other chips off board that I was evaluating and testing, so I am now using a custom made board with our peripherals on. This design might be released at some point, but I don't have plans to do so yet until the software is all written and working. It also uses the Garmin s340 softdevice (needed fro ANT+ and BLE on same device), which you must source yourself from Garmin. I cannot distribute this, so there is no way to make this compile 'out of the box'.

2) I am using NRF SDK directly and compiling with the arm embedded toolchain. It uses cmake as the build system, which I forked from this repo and modified to work with my board: https://github.com/NRB-Tech/nRF5-cmake-scripts
NRF SDK is Nordics C toolkit for programming their NRF52 line of chips (the actual SoC inside the seeed).

3) I am not using Arduino as it doesn't support the full range of NRF functionality and is incompatable with the ANT+ stack and softdevice I am using on the board, hence it will never be portable to Arduino IDE.

So in conclusion, check back in a few months and there might be something more concrete to look at. For now, it's just a place to host my project whilst I work on it.
